carts/src/routes/index.ts

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+import { videoGames } from "../models/games";
+
+const routes = express.Router();
+
+routes.get("/", (req, res) => {
+  res.json("Welcome to the store!");
+});
+
+routes.get("/cart-items", (req, res) => {
+  res.json(videoGames);
+});
```

Comment:
Would like to have seen you try to tackle the params. Here is my solution. Reach out if you have any questions:

```
  .get("/", (req: Request, res: Response) => {
    try {
      let filteredItems = items;
      if (req.query.maxprice) {
        const max = Number(req.query.maxprice);
        filteredItems = filteredItems.filter((item) => item.price > max);
        // return res.status(200).json(filteredItems);
      }

      if (req.query.prefix) {
        const prefix = String(req.query.prefix);
        filteredItems = filteredItems.filter((item) =>
          item.product.startsWith(prefix)
        );
      }

      if (req.query.limit) {
        console.log("limit called");
        const limit = Number(req.query.limit);
        filteredItems = filteredItems.slice(0, limit);
        console.log(filteredItems);
      }
      return res.status(200).json(filteredItems);
    } catch (e) {
      console.log(e);
      return res.status(500).json(e);
    }
  })
```